concepts/pointers/introduction.md

```diff
@@ -1,3 +1,149 @@
-# Introduction
+# About
+
+Like many other languages, Go has pointers. A pointer hold the memory address of a value.
```

Comment:
I think we need to give people a reason for why they're learning this before they work through such a complex topic. I've copied and shortened the bit at the bottom of document here. 

```suggestion
Like many other languages, Go has pointers. 
If you're new to pointers, they can feel a little mysterious but once you get used to them, they're quite straight-forward.
They're a crucial part of Go, so take some time to really understand them.

Before digging into the details, it's worth understanding the use of pointers. Pointers are a way to share memory with other parts of our program, which is useful for two major reasons:
1. When we have large amounts of data, making copies to pass between functions is very inefficient. 
  By passing the memory location of where the data is stored instead, we can dramatically reduce the resource-footprint of our programs.
2. By passing pointers between functions, we can access and modify the computer's memory directly, meaning that any changes made by one function are immediately visible to other parts of the program when the function ends.
```